index.html

```diff
@@ -8,7 +8,149 @@
+        как всегда помогает математика &#129299&#128520&#128513 Например с этой формулой: y=<sub>1</sub>&int;<sup>2</sup>x<sup>3</sup> &#128526
+    </p>
+
+   <hr>
+
+    <ol>
+        <li>Знакомство в Web
+            <ul>
+                <li>Краткая история Web и что такое интернет.</li>
+                <li>Как устроен браузер и как он получает страницы по сети.</li>
+                <li>Что такое Git?</li>
+                <li>Создание первого репозитория?</li>
+                <li>О файле .gitignore.</li>
+            </ul>
+        </li>
+        <li>HTML-верства - первые шаги
+            <ul>
+                <li>Настройка рабочей среды.</li>
+                <li>Краткая история HTML.</li>
+                <li>Какие бывают тэги и что такое атрибутю.</li>
+                <li>Тэги для работы с текстом.</li>
+                <li>Создание собственной страницы.</li>
+            </ul>
+        </li>
+        <li>Погружение в HTML-разметку
+            <ul>
+                <li>Мета-тэги.</li>
+                <li>Мнемоники.</li>
+                <li>Тег для работы с изображениями.</li>
+                <li>Теги для работы со списками.</li>
+                <li>Теги для работы с таблицами.</li>
+                <li>Теги для работы с формами.</li>
+                <li>Div-блоки.</li>
+            </ul>
+        </li>
+    </ol>
+    <hr>
+    <p>ТАБЛИЦА</p>
+    <table border="1">
+        <tr>
+            <th>Tag</th>
+            <th>Description</th>
+        </tr>
+        <tr>
+            <td><xmp><table></xmp></td>
+            <td>Defined a table</td>
+        </tr>
+        <tr>
+            <td><xmp><tr></xmp></td>
+            <td>Defines a row in a table</td>
+        </tr>
+        <tr>
+            <td><xmp><th></xmp></td>
+            <td rowspan="2">Defines a "header cell" / "cell" in a table</td>
+        </tr>
+        <tr>
+            <td><xmp><td></xmp></td>
+        </tr>
+        <tr>
+            <td  colspan="2"><xmp>Rarely used</xmp></td>
+        </tr>
+        <tr>
+            <td><xmp><caption></xmp></td>
+            <td>Defines a table caption</td>
+        </tr>
+        <tr>
+            <td><xmp><thead></xmp></td>
+            <td>Groups the header content in a table</td>
+        </tr>
+        <tr>
+            <td><xmp><tbody></xmp></td>
+            <td>Groups the body content in a table</td>
+        </tr>
+        <tr>
+            <td><xmp><tfoot></xmp></td>
+            <td>Groups the footer content in a table</td>
+        </tr>
+    </table>
+    <hr>
+    <p>ФОРМА</p>
+    <form action="/" method="get">
+        <fieldset>
+            <legend>My first form</legend>
+        <div><div><label>Name</label></div><input type="text" placeholder="Name" required></div>
+        <div><div><label>Email</label></div><input type="email" required></div>
+        <div><div><label>Password</label></div><input type="password" required></div>
```

Comment:
input doesn't understand that it has label cause you has 2 ways
1) wrap input into the label tag
2) set id to input and use attribute for into label : <label for=<id>> Label </label>